core/templates/domain/platform_feature/platform-feature-admin-backend-api.service.ts

```diff
@@ -35,11 +45,27 @@ export class PlatformFeatureAdminBackendApiService {
     private http: HttpClient,
   ) {}
 
+  async getFeatureFlags(): Promise<FeatureFlagsResponse> {
+    return new Promise((resolve, reject) => {
+      this.http.get<FeatureFlagsDicts>(
+        '/feature_flags').toPromise().then(response => {
+        resolve({
+          featureFlags: response.feature_flags.map(
+            dict => PlatformParameter.createFromBackendDict(
+              dict)
+          )
+        });
+      }, errorResponse => {
+        reject(errorResponse.error.error);
+      });
+    });
+  }
+
   async updateFeatureFlag(
       name: string, message: string, newRules: PlatformParameterRule[]):
       Promise<void> {
     await this.http.post(
-      AdminPageConstants.ADMIN_HANDLER_URL,
+      '/feature_flags',
```

Comment:
Should use a named constant here and elsewhere.

Reply:
Done